apps/meteor/client/views/admin/apps/AppDetailsPage/AppDetailsPageHeader.tsx

```diff
@@ -25,16 +37,36 @@ const AppDetailsPageHeader = ({ app }: { app: App }): ReactElement => {
 					{bundledIn && Boolean(bundledIn.length) && <BundleChips bundledIn={bundledIn} />}
 				</Box>
 				{app?.shortDescription && <Box mbe='x16'>{app.shortDescription}</Box>}
+
 				<Box display='flex' flexDirection='row' alignItems='center' mbe='x16'>
-					<AppStatus app={app} installed={installed} isAppDetailsPage={true} isSubscribed={isSubscribed} />
+					<AppStatus app={app} installed={installed} isAppDetailsPage={true} />
 					{(installed || isSubscribed) && <AppMenu app={app} mis='x8' />}
 				</Box>
 				<Box display='flex' flexDirection='row' color='hint' alignItems='center'>
 					<Box fontScale='p2m' mie='x16'>
 						{t('By_author', { author: author?.name })}
 					</Box>
 					<Box is='span'> | </Box>
-					<Box mi='x16'>{t('Version_version', { version })}</Box>
+
+					<Box marginInlineStart='16px' marginInlineEnd='4px'>
+						{t('Version_version', { version: versioni18nKey(app) })}
+					</Box>
+
+					<Box is={'span'} marginInlineEnd='x16' marginInlineStart='x4' marginBlockStart='x4'>
+						{versionIncompatible === true && (
+							<>
+								<TooltipOnHover
```

Comment:
You can remove this component and just add `title` prop in the component you want to display the tooltip